tests/test_smc.py

```diff
@@ -0,0 +1,164 @@
+from jax.config import config
+import pickle
+import os
+import time
+
+from md.states import CoordsVelBox
+
+config.update("jax_enable_x64", True)
+
+from md import enhanced
+from md.moves import NPTMove
+
+import numpy as np
+from tests import test_ligands
+
+import jax
+from timemachine.potentials import bonded
+
+from fe import functional
+from ff import Forcefield
+from ff.handlers.deserialize import deserialize_handlers
+
+# (ytz): useful for visualization, so please leave this comment here!
+import asciiplotlib as apl
```

Comment:
Add `asciiplotlib` to `requirements.txt`?

Reply:
Again, not really meant to be merged, but rather an illustrative example. will remove later once the we work out the test case properly.